src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -180,6 +181,8 @@ public class MicrobeEditor : NodeWithInput, ILoadableGameState, IGodotEarlyNodeR
 
     private bool microbePreviewMode;
 
+    private bool worldEntitiesVisible = true;
```

Comment:
This doesn't seem to be needed? The property could be get only as the set always sets the currently given `value` into the specified nodes, so this storage variable is not actually even used for anything.